### PR TITLE
Update UserGuide with UI overview and FindCommand information

### PR DESCRIPTION
**Pull request by sp4ce-cowboy**
- FindCommand Example and UI details added

### CODE REVIEW

**Review by jamz903 — DISMISSED** [review]
lgtm!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -620,6 +735,83 @@ Important notes:
 
 #### Find Transaction(s)
 
+Finds a `Transaction` in UniCa$h.
+
+Command: `find n/NAME l/LOCATION c/CATEGORY`
```
Comment (woojiahao) [inline docs/UserGuide.md:740]: Could you use the standardized notation for optional parameters: `[n/Name]`, thanks!

**Review by woojiahao — CHANGES_REQUESTED** [review]
Just a minor comment on the parameter format

File: docs/UserGuide.md (hunk 1 above)
Comment (sp4ce-cowboy) [reply docs/UserGuide.md:740]: Ok sure

**Comment by sp4ce-cowboy** [comment]
@woojiahao @jamz903 Found a filter remnant bug in the CommandType enum that I've fixed, also upated the image with the correct output.

**Review by woojiahao — APPROVED** [review]
LGTM!

**Pull request merged**